### ISSUE TITLE
Can't see the controls of iota wallet interface

### ISSUE THREAD

**Issue by luisfernandobr** [iotaledger/wallet#577]
I installed iota's wallet on linux (ubuntu 17.10) and I can not see any controls on its interface.
![screenshot from 2017-12-02 21-17-05](https://user-images.githubusercontent.com/617294/33520671-b0c7bea2-d7a6-11e7-9cba-66cfbc57c229.png)

**Comment by jstillings1** [comment]
This is simply because the wallet is not connecting to the node that is selected on install.
Using the tools menu. hit Edit node configuration
Select a different host from the drop down menu
You might have to pick a different host over and over until you find one that is actually up.
Remember there is 24 light wallet hosts and right now there is 1000's of new users installing the wallet and taking their first steps.

**Comment by wonkytonky** [comment]
or in some cases .. the node changing didnt work.. but there are ppl who had the same issue and there was a fix ..  might wanne look into closed/old issues if node changing doesn't help

**Comment by agilob** [comment]
@jstillings1 People in this thread claim it's because of poor OpenCL support https://github.com/iotaledger/wallet/issues/416

**Comment by nasaiya** [comment]
I think there is more to it than failure to connect to a node. I've got two linux boxes, both Gentoo, both similar setups, both set to connect to the same node, yet one works and the other does not.

I don't know enough about this to try and find the bug myself, but they are definitely both connecting to the node.

**Comment by agilob** [comment]
Definitely a problem with GPU drivers, it's all green on my laptop with AMD and Intel, works fine on PC with nvidia nouveau.

**Comment by heltonunesilva** [comment]
> Definitely a problem with GPU drivers, it's all green on my laptop with AMD and Intel, works fine on PC with nvidia nouveau. There was no activity on any branch from devs, that's worrying.

Dude, this is just me. There in my work I use a PC with NVIDIA graphics card running ubuntu 16.04, at home a notebook with intel graphics card running ubuntu 17.04, the first works fine, but on the laptop only green screen. sucks!

**Comment by nasaiya** [comment]
> There was no activity on any branch from devs, that's worrying.
Not really, they're probably focusing their efforts on the UCL wallet...

All the systems I have issues with are using the opensource ati drivers. Nvidia, and xorg dummy driver both work for me. I think it just doesn't like the ati drivers for some reason -- probably the same reason why I can't seem to use my ati card for mining without using the ancient proprietary ati drivers...
